database/migrations/2018_07_16_144151_CreateTeamTable.php

```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
```

Comment:
StreamSubscrbers is to map stream to users. We should create Team table separately here and not change the existing migration files.